### ISSUE TITLE
asyncio.Task parameters are incorrect (because outdated typeshed)

### ISSUE THREAD

**Issue by kevinderuijter** [astral-sh/ty#2299]
### Summary

Ty says: Argument `eager_start` does not match any known parameter of function `create_task`
However, the argument does exist but ty is looking at python 3.11 signature.

This bug can be reproduced in the UV playground.
The share button in [https://play.ty.dev](https://play.ty.dev) doesn't work for me so the code is below.

```py
import asyncio

async def connect():
    ...

async def main():
    # incorrect ty error about argument eager_start here.
    asyncio.create_task(connect(), eager_start=True)
```

See the tasks.pyi as the cause.

```py
if sys.version_info >= (3, 11):
    def create_task(coro: _CoroutineLike[_T], *, name: str | None = None, context: Context | None = None) -> Task[_T]:
        """Schedule the execution of a coroutine object in a spawn task.

        Return a Task object.
        """

else:
    def create_task(coro: _CoroutineLike[_T], *, name: str | None = None) -> Task[_T]:
        """Schedule the execution of a coroutine object in a spawn task.

        Return a Task object.
        """
```

eager_start has been available as a keyword argument since python 3.14: https://docs.python.org/3/library/asyncio-task.html#creating-tasks

### Version

ty 0.0.8

**Comment by kevinderuijter** [comment]
Issue has to do with typeshed, not ty.
https://github.com/python/typeshed/issues/15210
